library/item/bows/src/main/java/org/quiltmc/qsl/bows/api/BowExtensions.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 QuiltMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.quiltmc.qsl.bows.api;
+
+import net.minecraft.item.ItemStack;
+
+/**
+ * An interface to implement for all custom bows in Quilt. <br>
```

Comment:
Those `<br>` probably should be `<p>` instead.

Reply:
There are a couple in the item group API. but fixed these